### PR TITLE
Update e2e tests for Nutanix

### PR DESCRIPTION
**Pull request by thunderboltsid**
Add kubernetes 1.25 related e2e tests for Nutanix.

### CODE REVIEW

File: test/e2e/simpleflow_test.go [hunk 1]
```diff
@@ -615,3 +615,14 @@ func TestNutanixKubernetes124SimpleFlowWithUUID(t *testing.T) {
...
 	runSimpleFlow(test)
 }
+
+func TestNutanixKubernetes125SimpleFlowWithUUID(t *testing.T) {
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewNutanix(t, framework.WithUbuntu125NutanixUUID(),
+			framework.WithPrismElementClusterUUID(),
+			framework.WithSubnetUUID()),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube125)),
```
Comment (taneyland) [inline test/e2e/simpleflow_test.go:625]: We have to add this line to set the feature flag for 1.25 on all of the 1.25 e2e tests in this PR. Else it'll fail at the validation level.

```suggestion
		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube125)),
framework.WithEnvVar(features.K8s125SupportEnvVar, "true"),
```

**Review by taneyland — APPROVED** [review]
Thanks for adding these tests! Looks great 🙌

**Comment by thunderboltsid** [comment]
Seems like https://github.com/aws/eks-anywhere/pull/3475 has done significant refactoring leading to a merge conflict

**Comment by abhinavmpandey08** [comment]
1.25 image is built. Unholding

/unhold

**Review by abhinavmpandey08 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**